client/Cargo.toml

```diff
@@ -32,11 +32,12 @@ http = "0.2.8"
 rand = "0.8.5"
 serde = { version = "1.0.142", features = ["derive"] }
 serde_json = "1.0.83"
-tungstenite = "0.16"
+tungstenite = { version = "0.16", features = ["native-tls"] }
```

Comment:
Without that tungstenite won't work with wss protocol at all.
The other option is to use rust-tls.
I don't know which is better, but native-tls is more popular and, I believe, should be faster

Reply:
We already have `native-tls` enabled, (thanks to workspaces resolving features in a weird way), but making it explicit helps.

Reply:
Do you mean `iroha2-dev` specifically? Because it didn't work on lts branch until I enabled this feature

Reply:
Nope. Linux x86.